osu.Game/Screens/Play/Player.cs

```diff
@@ -672,7 +672,8 @@ public override void OnEntering(IScreen last)
                 .Delay(250)
                 .FadeIn(250);
 
-            Background.EnableUserDim.Value = true;
+            Background.EnableGameplayDim.Value = true;
+            Background.EnableUIBlur.Value = false;
```

Comment:
This feels all kinds of wrong. `Background` should probably expose a bindable of an enumeration type that's an actual exclusive choice, such as

```csharp
public enum DimMode
{
    Off,
    UI,
    Gameplay
}
```

and then adjust depending on that.